site2/website-next/src/pages/index.js

```diff
@@ -1,4 +1,5 @@
-import React, { useEffect, componentDidMount } from "react";
+import React, { useEffect } from "react";
+import SineWaves from "sine-waves";
```

Comment:
@Paul-TT It looks like you haven't merge the latest changes to your `PR`, please merge the `main` branch into your branch and then submit your `PR`